websocket-channel-2023.html

```diff
@@ -0,0 +1,487 @@
+                  </figure>
+
+                  <p>A successful response will contain a URL to the subscription API that can be used directly with a JavaScript client.</p>
+
+                  <figure id="notification-subscription-response" class="example listing" rel="schema:hasPart" resource="#notification-subscription-response">
+                    <p class="example-h"><span>Example</span>: Response from the Notification Subscription including the WebSocket subscription URL.</p>
+
+                    <pre about="#notification-subscription-response" property="schema:description" typeof="fabio:Script"><code>HTTP/2</code>
+<code>Content-Type: application/ld+json</code>
+<code></code>
+<code>{</code>
+<code>  "@context": "https://www.w3.org/ns/solid/notification/v1",</code>
+<code>  "type": "WebSocketChannel2023",</code>
+<code>  "topic": "https://storage.example/data/resource",</code>
+<code>  "receiveFrom": "wss://websocket.example/?auth=Ys3KiUq"</code>
+<code>}</code></pre>
+
+                    <figcaption property="schema:name">Response to the <code>POST</code> request including <code>type</code> and <code>receiveFrom</code>.</figcaption>
+                  </figure>
+
+                  <p>In JavaScript, a client can use the data in the response to establish a connection to the WebSocket hypermedia API.</p>
+
+                  <figure id="websocket-new" class="example listing" rel="schema:hasPart" resource="#websocket-new">
+                    <p class="example-h"><span>Example</span>: Establishing a new WebSocket connection.</p>
+
+                    <pre about="#websocket-new" property="schema:description" typeof="fabio:Script"><code>const ws = new WebSocket(receiveFrom, type);</code></pre>
+
+                    <figcaption property="schema:name">JavaScript code to establish a <code>new WebSocket</code> connection</figcaption>
+                  </figure>
+
+                  <p>A client will define how to handle events, especially the <code>onclose</code> and <code>onmessage</code> events.</p>
+
+                  <figure id="websocket-onclose-onmessage" class="example listing" rel="schema:hasPart" resource="#websocket-onclose-onmessage">
+                    <p class="example-h"><span>Example</span>: Handling WebSocket events after the connection is established.</p>
+
+                    <pre about="#websocket-onclose-onmessage" property="schema:description" typeof="fabio:Script"><code>ws.onclose(evt => reconnect(evt));</code>
+<code>ws.onmessage(evt => console.log("Message received: ", evt))</code></pre>
+
+                    <figcaption property="schema:name">JavaScript code to handle <code>onclose</code> and <code>onmessage</code> events</figcaption>
+                  </figure>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="authentication-authorization" inlist="" rel="schema:hasPart" resource="#authentication-authorization">
```

Comment:
Unless there are different requirements, this section should match/refer to notifications-protocol's authentication-authorization